shunting-yard-replicator/src/main/java/com/expediagroup/shuntingyard/replicator/exec/messaging/MessageReaderAdapter.java

```diff
@@ -121,9 +128,11 @@ private MetaStoreEvent map(ListenerEvent listenerEvent) {
       break;
     case ALTER_TABLE:
       AlterTableEvent alterTable = (AlterTableEvent) listenerEvent;
-      if (alterTable.getTableLocation() != null) {
-        if (alterTable.getTableLocation().equals(alterTable.getOldTableLocation())) {
-          builder.replicationMode(ReplicationMode.METADATA_UPDATE);
+      if (isPartitionedTable(alterTable.getDbName(), alterTable.getTableName())) {
```

Comment:
Not a big deal but maybe can we unnest one of these ifs? Might read a bit better.

Could do a check before `if (!isPartitionedTable(...)) { break; }`

Reply:
Hmm.. not sure. I have merged first and second `if` conditions into one. Looks better ?

Reply:
Could make a method out of that one, `if(isSameLocation(...))`? But yep I'm happy with that, looks better.